corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -299,10 +303,11 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                 .replace("{selected_cases}", caseId)
                 .replace("{case_id}", caseId);
             e.target.className += " disabled";
-            this.iconIframe(e, actionUrl, caseId);
+            this.iconIframe(e, actionUrl, this.model.get('id'));
```

Comment:
Does changing this cause a problem when `caseId` is set by `this.model.get('groupKey')` above?

Reply:
I am only using it to re-fetch the case. So yes, it cannot be the parents case id.